packages/panzer/mini-em/example/BlockPrec/CMakeLists.txt

```diff
@@ -292,12 +287,12 @@ IF (NOT Kokkos_ENABLE_CUDA AND ${PACKAGE_NAME}_ENABLE_ML AND MueLu_ENABLE_Epetra
     NUM_MPI_PROCS 1
     )
 
-  TRIBITS_ADD_TEST(
-    BlockPrec
-    NAME "Darcy_ML"
-    ARGS "--solver=ML --numTimeSteps=1 --linAlgebra=Epetra --inputFile=darcyHex.xml  --x-elements=15  --y-elements=15 --z-elements=15"
-    NUM_MPI_PROCS 4
-    )
+  # TRIBITS_ADD_TEST(
+  #   BlockPrec
+  #   NAME "Darcy_ML"
+  #   ARGS "--solver=ML --numTimeSteps=1 --linAlgebra=Epetra --inputFile=darcyHex.xml  --x-elements=15  --y-elements=15 --z-elements=15"
+  #   NUM_MPI_PROCS 4
+  #   )
```

Comment:
@cgcgcg - looks like the Darcy test is now disabled? Just wanted to make sure that was intentional.

Reply:
Yes, the refactored interpolations have some issues when used with Epetra, but I don't see that as a major issue.